src/plugins/vis_type_timeseries/public/application/components/panel_config/timeseries.tsx

```diff
@@ -127,6 +127,7 @@ export class TimeseriesPanelConfig extends Component<
       legend_position: 'right',
       show_grid: 1,
       tooltip_mode: 'show_all',
+      ignore_daylight_time: 0,
```

Comment:
nit: I have no idea who started to use `0 `and `1` for boolean variables, but think we should not continue doing it. I vote for using Boolean instead

Reply:
I agree with that